djangocms_admin_style/static/js/base-admin.js

```diff
@@ -42,6 +42,23 @@
             }
         }
     });
+
+    // Back to link
+    var button = document.getElementById('header-btn');
+
+    if (window.name !== '') {
+        var object = JSON.parse(window.name);
```

Comment:
remember that `window.name` can also be set by someone else. you need to wrap JSON.parse in try / catch